www/vercel.json

```diff
@@ -1,5 +1,10 @@
 {
   "redirects": [
+    {
```

Comment:
do we want a redirect for any hash-links from /procedures#xyz-input-xyz that you removed to /validators?

Reply:
Done